doc/manuals.jp/admin/install.md

```diff
@@ -16,69 +11,34 @@
 <a name="introduction"></a>
 ## イントロダクション
 
-推奨される手順は、公式の [Docker Hub の Orion docker コンテナ](https://hub.docker.com/r/fiware/orion/) を使用してインストールすることです。
+推奨される手順は、公式の [Docker Hub の Orion docker コンテナ](https://hub.docker.com/repository/docker/telefonicaiot/fiware-orion) を使用してインストールすることです。詳細については、[このドキュメント](https://github.com/telefonicaid/fiware-orion/blob/master/docker/README.md)を参照してください。
 
-ただし、Docker 化されたインフラストラクチャがない場合は、インストールすることをお勧めします。
-CentOS8.x で RPM パッケージを使用します。 ソースからのビルドに興味がある場合は、[このドキュメント](build_source.md)を確認してください。
+ただし、Docker 化されたインフラストラクチャがない場合は、ソースからビルドをインストールすることをお勧めします。
+[このドキュメント](build_source.md)を確認してください。
 
 <a name="requirements"></a>
 ## 要件
 
-* システムリソース : [これらの推奨事項](diagnosis.md#resource-availability)を参照
-* オペレーティング・システム : CentOS/RedHat。リファレンス・オペレーティング・システムは CentOS 8.3.2011 ですが、それ以降の CentOS/RedHat 8.x バージョンでも動作します
-* データベース : MongoDB は、Orion Context Broker をインストールするホストと同じホストで実行するか、ネットワーク経由でアクセス可能な別のホストで実行する必要があります。推奨される MongoDB のバージョンは 4.4 です。古いバージョンでは動作しますが、推奨しません
-* RPM の依存関係 (これらのパッケージのいくつかは、公式の CentOS/RedHat リポジトリにはありませんが、EPEL にあります。EPEL リポジトリを設定する必要があります。<http://fedoraproject.org/wiki/EPEL> を参照してください) :
-    * contextBroker パッケージ (必須) は次のパッケージに依存しています : libstdc++, boost-thread, boost-filesystem, gnutls, libgcrypt, libcurl, openssl, logrotate, libuuid
+Docker hub で公式の Orion docker コンテナを使用してインストールする場合は、次のものが必要です:
 
-<a name="installation"></a>
-## インストール
+* [Docker](https://docs.docker.com/engine/install/)
+* [Docker compose](https://docs.docker.com/compose/install/) (not strictly needed, but strongly recommended) 
```

Comment:
"(not strictly needed, but strongly recommended)" should be translated?

Reply:
I have overlooked it. I translated it.